x-pack/plugins/stack_connectors/server/connector_types/cases_webhook/schema.ts

```diff
@@ -71,6 +71,9 @@ export const ExecutorSubActionPushParamsSchema = schema.object({
   incident: schema.object({
     title: schema.string(),
     description: schema.nullable(schema.string()),
+    id: schema.nullable(schema.string()),
```

Comment:
nullable because I did not add these to the connector test form. Should I?

Reply:
This is correct. All new fields should be optional to avoid breaking changes in the APIs.